gradle/error-prone.gradle

```diff
@@ -1,50 +1,50 @@
 apply plugin: 'net.ltgt.errorprone'
 
 dependencies {
+	// errorprone('com.google.errorprone:error_prone_core:2.43.0') bug ErrorProneJavacPlugin has been compiled by a more recent version of the Java Runtime (class file version 65.0
```

Comment:
Also the bump to v.44 causing jdk compiling issue which seems strange considering patch scope.